pype/plugins/global/publish/integrate_new.py

```diff
@@ -561,10 +561,17 @@ def copy_file(self, src, dst):
         while True:
             try:
                 copyfile(src, dst)
-            except OSError as e:
-                self.log.critical("Cannot copy {} to {}".format(src, dst))
-                self.log.critical(e)
-                six.reraise(*sys.exc_info())
+            except (OSError, AttributeError) as e:
+                self.log.warning(e)
+                # try it again with shutil
+                import shutil
+                try:
+                    shutil.copyfile(src, dst)
+                    self.log.debug("Copying files with shutil...")
+                except (OSError, AttributeError) as e:
+                    self.log.critical("Cannot copy {} to {}".format(src, dst))
```

Comment:
I would rather add `exc_info=True` here and remove `self.log.critical(e)` to see full traceback instead of logging only error message. And `AttributeError` is not required here.

Reply:
Great I like it, could you write here an example of your solution, please?